orchestration/dagster_orchestration/hca_manage/validation.py

```diff
@@ -71,6 +98,7 @@ def run(arguments: Optional[list[str]] = None) -> None:
     well_known_dirs = {'/data', '/descriptors', '/links', '/metadata'}
     for dir in well_known_dirs:
         validate_directory(gs_bucket.prefix + dir, bucket)
+        validate_staging_area(gs_bucket.prefix + dir)
```

Comment:
This should be run on the input path outside of the loop, not the gs_bucket + well known dir

Reply:
Have refactored this to take in the args.path